lib/bulk_data_test_kit/v1.0.1/bulk_data_status_check_test.rb

```diff
@@ -22,6 +22,24 @@ class BulkDataStatusCheckTest < Inferno::Test
       DESCRIPTION
       # link 'http://hl7.org/fhir/uv/bulkdata/STU1.0.1/export/index.html#bulk-data-status-request'
 
+      verifies_requirements 'hl7.fhir.uv.bulkdata_2.0.0@123',
+                            'hl7.fhir.uv.bulkdata_2.0.0@125',
+                            'hl7.fhir.uv.bulkdata_2.0.0@132',
+                            'hl7.fhir.uv.bulkdata_2.0.0@144',
+                            'hl7.fhir.uv.bulkdata_2.0.0@145',
+                            'hl7.fhir.uv.bulkdata_2.0.0@147',
+                            'hl7.fhir.uv.bulkdata_2.0.0@150',
+                            'hl7.fhir.uv.bulkdata_2.0.0@153',
+                            'hl7.fhir.uv.bulkdata_2.0.0@154',
+                            'hl7.fhir.uv.bulkdata_2.0.0@155',
+                            'hl7.fhir.uv.bulkdata_2.0.0@156',
+                            'hl7.fhir.uv.bulkdata_2.0.0@157',
+                            'hl7.fhir.uv.bulkdata_2.0.0@159',
+                            'hl7.fhir.uv.bulkdata_2.0.0@162',
```

Comment:
Can we test 162? Does this test 162? How do we know if using OAuth 2.0 bearer tokens?

162
> Response - Complete Status:
> 
> `requiresAccessToken`: Value SHALL be true if both the file server and the FHIR API server control access using OAuth 2.0 bearer tokens.

Reply:
Does not test for 162, removed.